ephemeral-java-client/src/main/java/io/carbynestack/ephemeral/client/EphemeralClient.java

```diff
@@ -76,27 +76,25 @@ private EphemeralClient(
    * Triggers a program execution.
    *
    * @param activation {@link Activation} configuration used as input for the program activation.
-   * @return A future completing normally either with a list of Amphora secret identifiers or an
-   *     http error code if the execution failed on server side and exceptionally in case an error
-   *     occurs on the network or representation layer
+   * @return Either a list of Amphora secret identifiers or an http error code if the execution
+   *     failed on server side.
+   * @throws CsHttpClientException In case an error occurs on the network or representation layer
```

Comment:
What exactly do you mean? It returned a `Future` before the change which is no longer the case, which is reflected in the Javadoc. In case you mean the mismatch between "list of Amphora secret identifiers" and `ActivationResult` that has been there before. But maybe it is good time to fix that. If you mean anything else, please elaborate.

Reply:
I am sorry, the link was badly formatted but is now correct. The referenced lines show the javadoc for `EphemeralClient`
>```java
>/**
> * A client for the Carbyne Stack Ephemeral service.
> *
> * <p>The client interacts with a single ephemeral backend service in order to execute MPC programs.
> *
> * <p>The methods of this client are all asynchronous and return a {@link Future}. This future
> * completes normally with either the requested domain object or a http status code.
> *
>* <p>In case a low-level error occurs on the network or representation layer the future terminates
> * exceptionally.
> *
> * <p>The public API of this class is defensive, i.e., arguments are checked for validity and an
> * {@link IllegalArgumentException} is thrown in case of a contract violation.
 >*/
>@Slf4j
>public class EphemeralClient {
>```

The client only provides a single method and this one is no longer returning a `Future`. I was wondering why the import of the no longer used `import io.vavr.concurrent.Future` was not removed and this is the case because it is still referenced in the class description.

Reply:
I see. Will fix that.

Reply:
I would say the middle part, i.e.,

```Java 
* <p>The methods of this client are all asynchronous and return a {@link Future}. This future
* completes normally with either the requested domain object or a http status code.
*
* <p>In case a low-level error occurs on the network or representation layer the future terminates
* exceptionally.
```
is then completely obsolete. Any objections, removing it?

Reply:
Fine for me